src/main/resources/mavenPostgreSQL/pom.xml

```diff
@@ -0,0 +1,29 @@
+<project  xmlns="http://maven.apache.org/POM/4.0.0"  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
To co tutaj stworzyłeś to drugi pom, którego nie potrzebujemy. Mamy pom główny, w którym mogą znajdować się różne pluginy, ale nie jestem przekonany czy plugin będzie nam potrzebny.
Pierwszym krokiem jest aplikacja, która uruchomi liquibase bez żadnych pluginów.
Usuń zatem proszę tego poma i wyczyść ręcznie schemat bazy danych. Uruchom aplikację i zobacz czy wtedy nowa tabelka powstanie - prawdopodobnie nie ze względu na złą ścięzkę, w której trzymane są pliki dbcghangelog.xml - domyslnie powinno leżeć prawdpodobnie w src/main/resource ale to do sprawdzenia